src/vs/workbench/browser/parts/editor/editorActions.ts

```diff
@@ -1086,6 +1111,36 @@ export class MaximizeGroupAction extends Action2 {
 	}
 }
 
+export class UnmaximizeEditorGroupAction extends Action2 {
+
+	constructor() {
+		super({
+			id: EXIT_MAXIMIZE_EDITOR_GROUP,
+			title: { value: localize('UnmaximizeEditorGroup', "Unmaximize Editor Group"), original: 'Unmaximize Editor Group' },
+			f1: true,
+			category: Categories.View,
+			precondition: MaximizedEditorGroupContext,
+			keybinding: {
+				weight: KeybindingWeight.WorkbenchContrib,
+				primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KeyK, KeyMod.CtrlCmd | KeyCode.KeyM),
+			},
+			menu: {
+				id: MenuId.EditorTitle,
```

Comment:
Isn't this now double contributed also from `editor.contribution.ts`?

Reply:
In the Editor Title Menu Dropdown the user can maximize the editor group. So he should also be able to unmaximize it from there.
This contribution here is to add it to the navigation in the toggled state so that the user can clearly see that he is in the maximized state.